### PR TITLE
Draft: Meta README

### PR DESCRIPTION
**Pull request by pnadolny13**
I'm not sure if this fits here or https://github.com/MeltanoLabs/.github/blob/main/profile/README.md?

I took a stab at documenting the expectations, roles, permissions, maintenance status, contribution steps, etc. based on some of the comments from https://gitlab.com/meltano/meta/-/issues/73

TODO:
- I dont know how the current process works for migrating an existing connector to MeltanoLabs, I'd like to at least add an overview and direct people to submit and issue for and admin to do it.
- I couldnt find the issue where we decided on the Teams structure and what permissions are given to each role but this breakdown seemed right. Were we planning to create a team per repo like "tap-google-analytics-contributor" and "..-maintainer"

maintainer: maintain
contributor: write
community member (anyone): read
admin (internal): admin

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,93 @@
...
+
+If you'd like to create a new connector in MeltanoLabs create an issue labeled "Add New Connector - [your connector name]" in the [Meta repo](https://github.com/MeltanoLabs/Meta) and assign it to @tayloramurphy and @pnadolny13 when youre code is ready to be pushed up to GitHub and they will create you a repository and give you access.
+Alternatively you could build it in your own namespace and later migrate it once its ready.
+
+### Migrating An Existing Connector
+
+Its understandable that sometimes community members build new connector or create a fork and add features in their own namespace which then becomes the primary variant used by the community, but they no longer have the capacity or desire to maintain it.
+In this case the primary maintainer has the option to migrate their connector to MeltanoLabs where it can live and be maintained into the future.
+
+Connectors in MeltanoLabs have quaility expectations, if the connectors isn't there yet we will label it with the appropriate maintenance status in its current state and request community contributions to get it up to the Active (Stable) status we expect.
```
Comment (tayloramurphy) [inline README.md:56]:
```suggestion
Connectors in MeltanoLabs have quality expectations. If the connector does not meet the standard then we will label it with the appropriate maintenance status in its current state and request community contributions to get it up to the Active (Stable) status we expect.
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,93 @@
...
+1. If no one from the community responds in a timely manner, we can offer the PR author to become a contributor if they have a history of contributing to Meltano and/or other open source projects. (We can use their GitHub history to help in this evaluation process.)
+1. Any successfully-merged PR automatically promotes that person to "contributor" and optionally to maintainer if they've consented in step 3 and no other maintainers exist.
+
+
+## Maintenance Statuses
+
+Active (Stable): The connector repository is well maintained and recommended for use in production environments.
+The repository has all of the following attributes; all known issues and bugs are addressed promptly, changes are expected to go through pull requests reviews, releases to main are created on the appropriate cadence, security patches are addressed promptly, a CI pipeline that runs automated testing is present, community contributed pull requests are reviewed in a timely manner, README is complete with examples and detail, etc.
```
Comment (tayloramurphy) [inline README.md:80]:
```suggestion
The repository has all of the following attributes; all known issues and bugs are addressed promptly, changes are expected to go through pull request reviews, releases to main are created on the appropriate cadence, security patches are addressed promptly, a CI pipeline that runs automated testing is present, community contributed pull requests are reviewed in a timely manner, README is complete with examples and detail, etc.
```

**Review by tayloramurphy — APPROVED** [review]

**Comment by tayloramurphy** [comment]
LGTM @pnadolny13 - feel free to merge or we can wait for AJ's feedback. Either way, this is a great first iteration.

**Review by aaronsteers — APPROVED** [review]
@pnadolny13 - This looks great, a very nice iteration!

**Pull request merged**